src/frontend/src/lib/types/address.ts

```diff
@@ -1,3 +1,5 @@
-export type EthAddress = string;
+export type Address = string;
```

Comment:
Isn't this unrelated to the goal of this PR? -> Separate PR?

Reply:
ah wow, sorry, different PR, removing it